src/wrapper_array.jl

```diff
@@ -57,6 +57,7 @@ parent_type(::Type{<:NamedDimsArray{L,T,N,A}}) where {L,T,N,A} = A
 Base.parent(x::NamedDimsArray) = getfield(x, :data)
 
 dim(a::NamedDimsArray{L}, name) where {L} = dim(L, name)
+dim(a::AbstractArray, d) = d
```

Comment:
should this error if out of bounds?

Reply:
My feeling was it's simpler to let whatever this gets fed to check, or not. Some functions like `size(A,d)` accept any positive `d`.